geowebcache/core/src/main/java/org/geowebcache/config/FileBlobStoreInfo.java

```diff
@@ -31,12 +33,20 @@
  */
 public class FileBlobStoreInfo extends BlobStoreInfo {
 
+    public static enum PathGeneratorType {
+        DEFAULT,
+        TMS,
+        SLIPPY
```

Comment:
I would go with XYZ just to be a bit more self documenting

Reply:
Renamed